kafka-ui-react-app/.eslintrc.json

```diff
@@ -29,7 +30,8 @@
     "@typescript-eslint/explicit-module-boundary-types": "off",
     "jsx-a11y/label-has-associated-control": "off",
     "import/prefer-default-export": "off",
-    "@typescript-eslint/no-explicit-any": "error"
+    "@typescript-eslint/no-explicit-any": "error",
+    "react/jsx-props-no-spreading": "off"
```

Comment:
This will disable the rule for a whole project. Let's disable only for tests via `overrides` in eslint config

> `{
>   "files": [ "*.spec.tsx" ],
>   "rules": {
>     "react/jsx-props-no-spreading": "off"
>   }`